Add Yivi portal content (#103)
privacybydesign/irma-documentation: yivi-docs/docs/trusted-verifier.md

---
title: Trusted Verifier
---

> Simply put, Trusted Verifier is a seal of approval from Yivi that a verifier can obtain which will be shown to the user on their Yivi. 

Reviewer: ```suggestion
Simply put, Trusted Verifier is a seal of approval from Yivi that a verifier can obtain, which will be shown to users in the Yivi app. 
```

Only verifiers who are known to the Yivi app through <a href="https://github.com/privacybydesign/pbdf-requestors">requestors scheme</a> will be able to benefit from this. 
 Previously, joining this scheme was only possible through making a PR with the necessary information (name, id, hostname, and logo) set up in the required format. Now we can help you through this process possible via <a href="https://portal.yivi.app">Yivi Portal</a>, which will be a companion synced with changes in the IRMA schemes.

Trusted Verifier has the following goals:

- Provide users with guidance to make informed decisions and as a precaution against data oversharing, which is a critical privacy concern

> - Provide a user experience that is intuitive

Reviewer: ```suggestion
- Provide a user experience that is intuitive and feels secure
```

- Help keep verifiers accountable and share their verification costs

<div class="center-container">
  <img
    src="/img/trusted-verifier/trusted.png"
    style={{ width: "35%"}}
    alt="disclosure-flow"
  />
  <img
    src="/img/trusted-verifier/untrusted.png"
    style={{ width: "35%" }}
    alt="disclosure-flow"
  />
</div>

## Using Yivi Portal to become a Trusted Verifier

> To become a Trusted Verifier, we have created a portal for organizations. 

Reviewer: Add link to the portal?

The portal holds organizations that were already a part of the <a href="https://github.com/privacybydesign/pbdf-requestors">requestors scheme</a> as
well as making it possible to create new organizations. An organization in this
context is an entity in Yivi that can become a verifier (relying party) or an issuer (attestation provider). 

The steps to become a Trusted Verifier using the Yivi Portal are as follows:

> 1. Log in to the Portal with your email address. A user with your email address will be created.

Reviewer: Maybe good to mention you need the Yivi app with a verified emailaddress to login
